src/components/layout/nav/Navbar.scss

```diff
@@ -5,6 +5,23 @@ $navbarIndex: 102;
 $hamburgerIconIndex: 103;
 $closeSignUpIconIndex: 105;
 
+body a {
```

Comment:
this affects all links in the site, is that intended? i don't think all links should be black generally 

https://ux.stackexchange.com/questions/2278/should-hyperlinks-be-blue/

Reply:
it was intended actually because those are the nav links and we don't have anchors.

Reply:
right but `body a` effects more than the navlinks, it affects all anchors in the whole website (unless this is somehow scoped only to the navbar? which i don't think it is)